### PR TITLE
Fixed issue #192: Added the FAQ.rst file that answers this issue

### PR DESCRIPTION
**Pull request by Devyani-Divs**
@pypingou Added the FAQ.rst file that answers how to add comaintainers for a new package request. Please let me know if this needs work. Thanks :)

### CODE REVIEW

File: FAQ.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+How do I...
+===========
+
+...specify comaintainers on a new package request?
+--------------------------------------------------
+
+Once the package is created you can add other packagers, pseudo-users or groups:
+
+    1. Go to one of your package: https://admin.fedoraproject.org/pkgdb/package/apache-commons-chain/
```
Comment (pypingou) [inline FAQ.rst]: Maybe change the package name to something else ? For example ``<package name>`` ?

File: FAQ.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    2. Click on: Manage the committers/watchers/admins (any of these)
```
Comment (pypingou) [inline FAQ.rst]: Maybe we could list all the buttons here since this is the official doc :)

File: FAQ.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    3. There is then an Add someone button you can use.
```
Comment (pypingou) [inline FAQ.rst]: Should we put emphasis on the button, like around ``Add``

**Comment by pypingou** [comment]
The file is right but I would put it withing the doc itself, see the ``doc`` folder and add it to the index.rst so that it shows up when building the doc :)

**Comment by pypingou** [comment]
I was thinking of putting it in a different page, but maybe the front-page is fine

**Comment by Devyani-Divs** [comment]
@pypingou different page as in, linking to FAQ.rst in the index file? Is it?

**Comment by pypingou** [comment]
That was my original idea, but I'm pondering about it now, index page might be an option (especially for now, as the FAQ is short)

**Comment by Devyani-Divs** [comment]
@pypingou: cool then :)

**Comment by pypingou** [comment]
Let's go with this and we can always revisit later.

Thanks for your patch!

**Pull request merged**